po/pt_BR.po

```diff
@@ -90,7 +98,7 @@ msgstr "<th>Tamanho:</th>"
 
 #: ../src/window.c
 msgid "A GTK+ only lightweight archive manager"
-msgstr "Um gerenciador de pacotes leve baseado no GTK+"
+msgstr "O ‘Xarchiver’ é um ‘gerenciador de pacotes’, que também é conhecido por ‘gerenciador de compactação’ ou ‘compactador e descompactador de arquivos’, que permite criar, adicionar, extrair, remover e modificar os arquivos e pastas nos pacotes de vários formatos (7z, ar, arj, bzip2, gzip, lha, lzma, lz4, lzip, lrzip, lzop, rar, tar, tar.bz2, tar.gz, tar.lzma, xz, zip, zstd, deb e rpm), trata-se de um programa leve que é baseado na linguagem de programação GTK+ e é independente do tipo do ambiente de área de trabalho para a manipulação dos arquivos"
```

Comment:
This is a short description for the info box. Why do you think you have to write a novel here?